app/controllers/admin/works_controller.rb

```diff
@@ -113,6 +113,51 @@ def download_ohms_xml
       :disposition => ContentDisposition.format(disposition: "attachment", filename: "#{@work.oral_history_content!.ohms_xml.accession}.xml")
   end
 
+  # Biographical metadata form
+  # GET "/admin/works/ab2323ac/oh_bio_form"
+  def oh_biography_form
+    @work.oral_history_content!
+    @work.oral_history_content.
+      interviewee_birth ||= OralHistoryContent::DateAndPlace.new
+    @work.oral_history_content.
+      interviewee_death ||= OralHistoryContent::DateAndPlace.new
+    render :oh_biography_form
+  end
+
+  # PATCH/PUT /admin/works/ab2323ac/submit_oh_bio
+  def submit_oh_biography
+    oral_history_content = @work.oral_history_content!
+    oral_history_params = params['oral_history_content']
+    oral_history_content.interviewee_birth = new_item_from_params( nil,
+      oral_history_params["interviewee_birth_attributes"],
+      OralHistoryContent::DateAndPlace)
+    oral_history_content.interviewee_death = new_item_from_params( nil,
```

Comment:
This is getting a little bit simpler than last time, but i still think there should be a way easier way to do this, without that `new_item_from_params` that looks more like:

    oral_history_content.interviewee_birth = OralHistoryContent::DateAndPlace.new(oral_history_params["interviewee_birth_attributes"])

Or actually it should be even simpler than that, it should look mostly like:

    oral_history_content.update_attributes(params['oral_history_content'])

It is admittedly confusing how Rails does this stuff, and then made more confusing with our attr_json additions. But I think we should do it the fewer lines of code way. It will also need Rails "Strong Params", have you ever looked at or learned about those? 

That might be the missing piece. 

But the way we do this with the repeated fields in Work and the analagous form there should be the guide. There are repeated multi-field sub-models there; there is no need for a `new_item_from_params` method there. We should try to do it here similarly. 

It admittedly kind of confusing; I can take a stab at it if you like.

Reply:
Thanks for your help on Tuesday!